### PR TITLE
Adapt all occurences of SuSEfirewall2 to firewalld

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/28726

### CODE REVIEW

**Comment by okurz** [comment]
@SergioAtSUSE WDYT?

@rwx788 WDYT about all the occurences of SuSEfirewall2 in the autoyast profiles?

File: tests/console/wicked_before_test.pm [hunk 1]
```diff
@@ -13,13 +13,13 @@
 use base "consoletest";
 use strict;
 use testapi;
-use utils qw(systemctl setup_static_network is_sle sle_version_at_least);
+use utils qw(firewall systemctl setup_static_network is_sle sle_version_at_least);
```
Comment (SergioAtSUSE) [inline tests/console/wicked_before_test.pm]: Travis is complaining because the function is not there.

You put it on opensusebasetest.

**Comment by rwx788** [comment]
@okurz when I've adjusted profiles for sle 15 I've removed/replaced all SuSEfirewall2 occurrences. As of now it's still planned to make firewalld to work with same settings, so no reconfiguration should be necessary.

**Comment by okurz** [comment]
ready

**Comment by okurz** [comment]
completed, reworked and rebased. Please review and merge if possible

File: lib/opensusebasetest.pm [hunk 2]
```diff
@@ -482,6 +483,10 @@ sub enter_test_text {
     }
 }
 
+sub firewall {
+    return ((is_sle && !sle_version_at_least('15')) || (is_leap && !leap_version_at_least('15.0'))) ? 'SuSEfirewall2' : 'firewalld';
```
Comment (rwx788) [inline lib/opensusebasetest.pm]: Unfortunately PR was there for too long, so firewalld is now in TW as well. But as a good part, by virtue of good design, I guess it should be enough to adjust code here =)

Comment (okurz) [reply lib/opensusebasetest.pm]: That's already incorporated. The if checks for *older* versions of SLE and Leap now and falls back to SuSEfirewall2, basically making firewallctl the new default now as I also recommended to @DimStar77

Comment (rwx788) [reply lib/opensusebasetest.pm]: You're right, sorry. Please, fix tidy and I'll merge: 
qw should be used as function at line 18, column 19.  use MODULE 'func' for single imports.  (Severity: 4)

**Pull request merged**